THIRD-PARTY-LICENSES.csv

```diff
@@ -445,6 +448,38 @@
 "cytoscape-dagre@2.4.0","MIT","https://github.com/cytoscape/cytoscape.js-dagre"
 "cytoscape-popper@2.0.0","MIT","https://github.com/cytoscape/cytoscape.js-popper"
 "cytoscape@3.21.0","MIT","https://github.com/cytoscape/cytoscape.js"
+"d3-array@3.2.1","ISC","https://github.com/d3/d3-array"
```

Comment:
BTW, that's a large number of extra dependencies. 
Did they (ngx-markdown) happen to mention what happened?

Reply:
It looks like they added new integrations and features when they moved from version 13.x.x to 14. https://github.com/jfcere/ngx-markdown/releases/tag/v14.0.0